### PR TITLE
Replaced all the Toasts with Snackbar

### PR DESCRIPTION
**Pull request by Smarshal21**
Replaced all the Toasts with Snackbars.

### 🏁 Checklist

- [x] ⛑️ Tests (unit and/or integration) are included or not needed
- [x] 🔖 Capability is checked or not needed 
- [x] 📅 Milestone is set
- [x] 🌸 PR title is meaningful (if it should be in the changelog: is it meaningful to users?)

### CODE REVIEW

**Comment by Smarshal21** [comment]
@AndyScherzinger could you please review this

File: app/src/main/java/com/nextcloud/talk/jobs/NotificationWorker.kt [hunk 1]
```diff
@@ -734,6 +739,20 @@ class NotificationWorker(context: Context, workerParams: WorkerParameters) : Wor
 
     private fun sendNotification(notificationId: Int, notification: Notification) {
         Log.d(TAG, "show notification with id $notificationId")
+        if (ActivityCompat.checkSelfPermission(
+                applicationContext,
+                Manifest.permission.POST_NOTIFICATIONS
+            ) != PackageManager.PERMISSION_GRANTED
+        ) {
+            // TODO: Consider calling
```
Comment (mahibi) [inline app/src/main/java/com/nextcloud/talk/jobs/NotificationWorker.kt:747]: :+1:

Comment (mahibi) [reply app/src/main/java/com/nextcloud/talk/jobs/NotificationWorker.kt:747]: nice catch @Smarshal21
if you like, feel free to implement in follow up PR's

**Review by mahibi — APPROVED** [review]

**Comment by mahibi** [comment]
@Smarshal21  you were invited to the nextcloud android team so now you should e.g. be able to create pull requests right inside the nextcloud repo if you would like to

**Comment by mahibi** [comment]
@Smarshal21 i saw you already posted as guest on our nextcloud org instance.
If you like to have an account there, just ping me there to let me know your email address so i can invite you.

**Comment by AndyScherzinger** [comment]
@mahibi I did it without asking :S Sorry if that isn't okay for you @Smarshal21 🙏

**Comment by Smarshal21** [comment]
> @Smarshal21 i saw you already posted as guest on our nextcloud org instance. If you like to have an account there, just ping me there to let me know your email address so i can invite you.

Sure, samanwith21@gmail.com

**Comment by Smarshal21** [comment]
> @mahibi I did it without asking :S Sorry if that isn't okay for you @Smarshal21 🙏

No worries 😀

**Pull request merged**